### ISSUE TITLE
Cannot debug project as created by the UI.Next Client Server template

### ISSUE THREAD

**Issue by panesofglass** [intellifactory/monodevelop.websharper#13]
This is what I see when I try to run the project:

<img width="1278" alt="mono-xsp-ui-next-client-server" src="https://cloud.githubusercontent.com/assets/8803/9071191/97ebfcea-3ab9-11e5-87d7-082ceb585f5e.png">

Full stack trace:
```
  at (wrapper managed-to-native) System.Reflection.Module:InternalGetTypes (System.Reflection.Module)
  at System.Reflection.MonoModule.GetTypes () [0x00000] in /private/tmp/source-mono-mac-4.0.0-branch-c5sr3/bockbuild-mono-4.0.0-branch/profiles/mono-mac-xamarin/build-root/mono-4.0.3/mcs/class/corlib/System.Reflection/MonoModule.cs:296 
  at WebSharper.Sitelets.SiteLoading+TryLoadSiteB@51-2.Invoke (System.Reflection.Module m) [0x00002] in <filename unknown>:0 
```

**Comment by azolotko** [comment]
@panesofglass I had the same issue. Adding WebSharper.Compiler (NuGet) package solved it.

**Comment by panesofglass** [comment]
Thanks, @azolotko!

**Comment by Tarmil** [comment]
I just fixed this issue in intellifactory/websharper#471, the fix will be available on the next release which should come soon.
